staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -85,7 +85,7 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	groupVersions        = []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion}
+	groupVersions        = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
```

Comment:
Sounds good for now. @liggitt's proposal with the preferredVerison seems reasonable. We have to wire through that version here then. Follow-up is fine if we go that route.